tests/test_lowlevel.py

```diff
@@ -17,3 +21,36 @@ def test_chunked_upload():
 
     assert r.status_code == 200
     assert r.request.headers['Transfer-Encoding'] == 'chunked'
+
+
+_schemes_by_var_prefix = {
+    'http': ['http'],
+    'https': ['https'],
+    'all': ['http', 'https'],
+}
```

Comment:
There's nothing gained by this being a dictionary because we never do a lookup in it. Mind changing to a list of two-tuples and then remove the call to `.items` lower down?